packages/api/lib/writeRecords/write-granules.js

```diff
@@ -462,13 +466,22 @@ const _writeGranuleRecords = async (params) => {
         executionCumulusId,
         trx,
         granulePgModel,
+        writeConstraints,
       });
-      await granuleModel.storeGranule(apiGranuleRecord);
+
+      // Future: refactor to cover the entire object?
+      // Ensure PG default createdAt value is propagated to DynamoDB/ES
+      // in the case where _writeGranule is called without createdAt set
+      if (!apiGranuleRecord.createdAt) {
+        apiGranuleRecord.createdAt = pgGranule.created_at.getTime();
+      }
```

Comment:
I think this is right. So we're saying that if a write request comes in with a null or undefined `createdAt`, we'll use postgres as the source of truth because it will definitely have a timestamp set due to PG defaults.

Reply:
Yes, this.    It shouldn't happen given our current business logic, but given the exported methods it's possible.